src/api/goods.ts

```diff
@@ -10,10 +10,16 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
+      const sortedGoods = [...goods];
+
+      sortedGoods.sort((a, b) => a.name.localeCompare(b.name)).splice(5);
```

Comment:
1. Use slice instead of splice. It's faster method.
2. If you chain method calls - put them on the same indentation level

```suggestion
      sortedGoods
        .sort((a, b) => a.name.localeCompare(b.name))
        .slice(0, 5);
```

Reply:
The Linter didn't allow for the same indentation level nevertheless I moved it to the next line.